chatsky/llm/methods.py

```diff
@@ -0,0 +1,82 @@
+"""
+LLM methods
+-----------
+This module provides basic methods to support LLM conditions.
+These methods return bool values based on LLM result.
+"""
+
+import abc
+
+from pydantic import BaseModel
+
+from chatsky.core.context import Context
+from chatsky.llm._langchain_imports import LLMResult
+
+
+class BaseMethod(BaseModel, abc.ABC):
+    """
+    Base class to evaluate models response as condition.
+    """
+
+    @abc.abstractmethod
+    async def __call__(self, ctx: Context, model_result: LLMResult) -> bool:
+        """
+        Take Context and LLMResult as an input and return boolean.
```

Comment:
That's signature, not method description.
It should be something like "Determine if result of an llm satisfies the condition of this method."